docs/basics/roadmap-and-links.md

```diff
@@ -4,33 +4,26 @@
 
 Despite being a small team, we're incredibly agile and committed to delivering a steady stream of fresh and exciting features to our users. We operate on a fortnightly release cycle for substantial feature updates, ensuring that Webstudio is always evolving and improving.
 
-Transparency is one of our core values, and we believe in sharing our development journey with our community. You can follow along with the design, development, marketing and operations behind the scenes at Webstudio.
+Transparency is one of our core values, and we believe in sharing our development journey with our community. Follow along with the design, development, marketing, and operations behind the scenes at Webstudio.
 
 [**View Roadmap**](https://wstd.us/roadmap)
 
-<figure><img src="../.gitbook/assets/image (1).png" alt=""><figcaption><p>Roadmap at <a href="https://wstd.us/roadmap">https://wstd.us/roadmap</a></p></figcaption></figure>
```

Comment:
@itsNintu I think its a bad idea to include a screenshot of a roadmap, because it needs updating and will always be wrong

Reply:
Yeah static is not ideal. It is something I want to highlight on the page as a method to drive users to roadmap (just some text a button isn't super appealing...) 

I've replaced it with a dub link which generates a nicer/bigger clickable element and removed the image.